app/src/androidTest/java/com/github/houseorganizer/houseorganizer/MainScreenActivityTest.java

```diff
@@ -151,6 +159,17 @@ public void cycleIsDisplayed() {
         onView(withId(R.id.calendar_view_change)).check(matches(isDisplayed()));
     }
 
+    @Test
+    public void addDialog() throws ExecutionException, InterruptedException {
+        Intents.init();
+        onView(withText("Add household")).perform(click());
+        intended(hasComponent(CreateHouseholdActivity.class.getName()));
+        Intents.release();
+
+        auth.signOut();
+        FirebaseTestsHelper.signInTestUserWithCredentials(FirebaseTestsHelper.TEST_USERS_EMAILS[0], FirebaseTestsHelper.TEST_USERS_PWD[0]);
```

Comment:
Same here after this test the connected user is not the same as at the beginning. But as the tests are done in a random order we cannot know which user is connected